hw/js-lesson-2/scripts.js

```diff
@@ -0,0 +1,111 @@
+// 1 Запросить у пользователя его возраст и определить, кем он является: ребенком (0–2), подростком (12–18), взрослым (18_60) или пенсионером (60– ...).
+
+let age = +prompt("enter your age");
+
+if (age >= 0 && age <= 2) {
+    alert("you are child")
+} else if (age >=12 && age <= 18) {
+    alert("you are teenager")
+} else if (age >= 18 && age < 60) {
+    alert("you are adult")
+} else if (age >= 60) {
+    alert("you are retired")
+} else {
+    alert("thank you! go to another question")
+}
+// 2 Запросить у пользователя число от 0 до 9 и вывести ему спецсимвол, который расположен на этой клавише (1–!, 2–@, 3–# и т. д).
+
+let number = +prompt("enter number from 0 to 9");
+
+switch(number) {
+    case 0: alert(")");
+       break;
+    case 1: alert("!");
+        break;
+    case 2: alert("@");
+        break;
+    case 3: alert("#");
+        break;
+    case 4: alert("$");
+        break;
+    case 5: alert("%");
+        break;
+    case 6: alert("^");
+        break;
+    case 7: alert("&");
+        break;
+    case 8: alert("*");
+        break;
+    case 9: alert("(");
+        break;
+}
+
+// 3 Запросить у пользователя трехзначное число и проверить, есть ли в нем одинаковые цифры.
+let nmbr = prompt("enter number from 100 to 999");
+nmbr = "nmbr";
```

Comment:
what you are trying to do here?

Reply:
doesn't matter what user will enter in the prompt you will rewrite nmbr varibale to nmbr